tests/SmartContract.spec.ts

```diff
@@ -63,6 +63,15 @@ describe('Smart Contract', () => {
             await adminContract.revokeTrustedManagerPermission(user.address);
         });
 
+        test('has trusted manager permission', async () => {
+            await adminContract.grantTrustedManagerPermission(user.address);
+            const hasCMPermission = await adminContract.hasPermission(
+                user.address,
+                PermissionType.CLUSTER_MANAGER_TRUSTED_BY,
+            );
+            expect(hasCMPermission).toEqual('true');
```

Comment:
It also will be nice to convert 'true'/'false' to boolean inside the `hasPermission` method to make the result API more obviouse

Reply:
Just checked that the contract method `hasPermission` returns `boolean`, so it is not clear why you are checking it to equal `true` string?

Maybe:
```suggestion
            expect(hasCMPermission).toEqual(true);
```